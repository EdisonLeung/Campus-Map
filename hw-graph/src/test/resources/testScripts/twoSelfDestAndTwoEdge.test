# Creates 2 Nodes and Tests adding an edge on each node that references the node it
# came from. Also tests adding 2 edges from the first node to the second node

# Create graph
CreateGraph fourEdges

# Add 2 nodes
AddNode fourEdges a
AddNode fourEdges b

# Add 2 self referencing edges
AddEdge fourEdges a a AA
AddEdge fourEdges b b BB

# Add 2 edges from a to b
AddEdge fourEdges a b AB
AddEdge fourEdges a b AB1

# Print nodes and children
ListNodes fourEdges
ListChildren fourEdges a
ListChildren fourEdges b