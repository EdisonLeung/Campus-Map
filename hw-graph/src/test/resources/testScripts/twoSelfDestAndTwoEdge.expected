# Creates 2 Nodes and Tests adding an edge on each node that references the node it
# came from. Also tests adding 2 edges from the first node to the second node

# Create graph
created graph fourEdges

# Add 2 nodes
added node a to fourEdges
added node b to fourEdges

# Add 2 self referencing edges
added edge AA from a to a in fourEdges
added edge BB from b to b in fourEdges

# Add 2 edges from a to b
added edge AB from a to b in fourEdges
added edge AB1 from a to b in fourEdges

# Print nodes and children
fourEdges contains: a b
the children of a in fourEdges are: a(AA) b(AB) b(AB1)
the children of b in fourEdges are: b(BB)